views/templates/admin/AdminSeur.tpl

```diff
@@ -0,0 +1,336 @@
+													<option value="{$key|escape:'htmlall':'UTF-8'}">{$seur_order_state|escape:'htmlall':'UTF-8'}</option>
+												{/foreach}
+												</select>
+											</td>
+											<td>
+												<input type="submit" value="{l s='Filter' name='submitFilter' mod='seur'}" id="submitFilter" class="filter" />
+											</td>
+										</tr>
+									</thead>
+									{if $errors}									
+										<div class="bootstrap">
+											<div class="module_error alert alert-danger">
+												<button data-dismiss="alert" class="close" type="button">�</button>
+												{$errors|escape:'htmlall':'UTF-8'}
+											</div>
+										</div>
+									{/if}
+					<tbody>
+
+						<tr class="bold">
+					{if $headers}
+					{foreach $headers as $key => $header}
+						<th {if $key eq 'delivery' || $key eq 'details' } colspan="2" {/if} )>{$header|escape:'htmlall':'UTF-8'}</th>
+					{/foreach}
+					</tr>
+					{/if}
+					
+					{if $deliveries_data}
+					{foreach $deliveries_data item=delivery_data name='delivery_item'}
+						<tr {if $smarty.foreach.delivery_item.iteration % 2 != 0} class="alternate" {/if} >
+						{assign var='delivered' value=false}
+						{if $delivery_data}
+						{foreach $delivery_data as $key => $delivery_value}
+							{if $key eq 'Expedicion'}
+								{assign var='delivery_number' value=$delivery_value}
+								
+							{/if}
+							<td class='{$key}' {if $key eq 'EXPEDICION' || $key eq 'Detalles'} colspan='2' {/if}> 
+							{if !in_array($key, $headersOcultas)} {$delivery_value} {/if}
+							
+							{if $key eq 'Descripcion' && $delivery_value eq 'ENTREGA EFECTUADA'}
+								{assign var='delivered' value=true}
+							{/if}
+							{if $key eq 'EXPEDICION' && ($countryTo eq 'ES' || $countryTo eq '-' || $countryTo eq '') && $delivered}
+								<a href="/modules/seur/ajax/createDeliveryNote.php?back={$back|escape:'htmlall':'UTF-8'}&amp;token={Tools::getValue('token')}&expedition_number={$delivery_value|escape:'htmlall':'UTF-8'}&amp;token={$token|escape:'htmlall':'UTF-8'}&id_employee={$id_employee|escape:'htmlall':'UTF-8'}">
+										<img src="{$img_dir|escape:'htmlall':'UTF-8'}/png_ico.png" alt="{l s='Delivery' mod='seur'}" title="{l s='Delivery' mod='seur'}" />
+								</a>
+							{/if}
+							 {if $key eq 'Detalles'}
+								<a class="verDetalles" href="/modules/seur/ajax/getExpeditionAjax.php?expedition_number={$delivery_number|escape:'htmlall':'UTF-8'}&token={$token|escape:'htmlall':'UTF-8'}&id_employee={$id_employee|escape:'htmlall':'UTF-8'}">
+										<img src="{$img_dir|escape:'htmlall':'UTF-8'}/details.png" alt="{l s='See details' mod='seur'}" title="{l s='See details' mod='seur'}" />
+									</a>
+							{/if}
+							</td>
+							
+						{/foreach}
+						{/if}
+						</tr>
+					
+					{foreachelse}
+					<div class="bootstrap">
+						<div class="module_error alert alert-danger">
+							<button data-dismiss="alert" class="close" type="button">�</button>
```

Comment:
This file is not encoded in UTF-8. A character in this line is not properly displayed.